internal/pfcpsim/helpers.go

```diff
@@ -71,3 +77,53 @@ func getLocalAddress(interfaceName string) (net.IP, error) {
 
 	return nil, pfcpsim.NewNoValidInterfaceError()
 }
+
+// parseAppFilter parses an application filter. Returns a tuple formed by a formatted SDF filter
+// and a uint8 representing the Application QER gate status. Returns error if fail occurs while validating the filter string.
+func parseAppFilter(filter string) (string, uint8, error) {
```

Comment:
it would be nice to see a unit test for this function, but we can add it later since Jon is waiting for this change.

Reply:
Added